unearthed-arcana/planeshift/kaladesh/race-aetherborn.xml

```diff
@@ -1,142 +1,146 @@
-        </rules>
-    </element>
-    <!-- <element name="Charisma" type="Ability Score Improvement" source="Plane Shift: Kaladesh" id="ID_WOTC_PSK_ASI_AETHERBORN_CHARISMA">
-        <compendium display="false" />
-        <supports>Ability Score Improvement</supports>
-        <description>
-            <p>Your Charisma increases by 1.</p>
-        </description>
-        <sheet display="false" />
-        <rules>
-            <stat name="charisma" value="1"/>
-        </rules>
-    </element> -->
+	<element name="Strength" type="Ability Score Improvement" source="Plane Shift: Kaladesh" id="ID_WOTC_PSK_ASI_AETHERBORN_STRENGTH">
+		<compendium display="false" />
+		<supports>Ability Score Improvement, Aetherborn</supports>
+		<description>
+			<p>Your Strength increases by 1.</p>
+		</description>
+		<sheet display="false" />
+		<rules>
+			<stat name="strength" value="1"/>
+		</rules>
+	</element>
+	<element name="Dexterity" type="Ability Score Improvement" source="Plane Shift: Kaladesh" id="ID_WOTC_PSK_ASI_AETHERBORN_DEXTERITY">
+		<compendium display="false" />
+		<supports>Ability Score Improvement, Aetherborn</supports>
+		<description>
+			<p>Your Dexterity increases by 1.</p>
+		</description>
+		<sheet display="false" />
+		<rules>
+			<stat name="dexterity" value="1"/>
+		</rules>
+	</element>
+	<element name="Constitution" type="Ability Score Improvement" source="Plane Shift: Kaladesh" id="ID_WOTC_PSK_ASI_AETHERBORN_CONSTITUTION">
+		<compendium display="false" />
+		<supports>Ability Score Improvement, Aetherborn</supports>
+		<description>
+			<p>Your Constitution increases by 1.</p>
+		</description>
+		<sheet display="false" />
+		<rules>
+			<stat name="constitution" value="1"/>
+		</rules>
+	</element>
+	<element name="Intelligence" type="Ability Score Improvement" source="Plane Shift: Kaladesh" id="ID_WOTC_PSK_ASI_AETHERBORN_INTELLIGENCE">
+		<compendium display="false" />
+		<supports>Ability Score Improvement, Aetherborn</supports>
+		<description>
+			<p>Your Intelligence increases by 1.</p>
+		</description>
+		<sheet display="false" />
+		<rules>
+			<stat name="intelligence" value="1"/>
+		</rules>
+	</element>
+	<element name="Wisdom" type="Ability Score Improvement" source="Plane Shift: Kaladesh" id="ID_WOTC_PSK_ASI_AETHERBORN_WISDOM">
+		<compendium display="false" />
+		<supports>Ability Score Improvement, Aetherborn</supports>
+		<description>
+			<p>Your Wisdom increases by 1.</p>
+		</description>
+		<sheet display="false" />
+		<rules>
+			<stat name="wisdom" value="1"/>
+		</rules>
+	</element>
+	<!-- <element name="Charisma" type="Ability Score Improvement" source="Plane Shift: Kaladesh" id="ID_WOTC_PSK_ASI_AETHERBORN_CHARISMA">
+		<compendium display="false" />
+		<supports>Ability Score Improvement</supports>
+		<description>
+			<p>Your Charisma increases by 1.</p>
+		</description>
+		<sheet display="false" />
+		<rules>
+			<stat name="charisma" value="1"/>
+		</rules>
+	</element> -->
```

Comment:
i'm guessing it's more of a question for @Azzzzzy, but what's the point in this comment?